docs/en_US/getting-started/getting-started.md

```diff
@@ -1,113 +1,214 @@
-      apiVersion: apps.emqx.io/v2alpha1
-      kind: EMQX
-      metadata:
-        name: emqx
-      spec:
-        image: emqx/emqx:5.0.9
-    EOF
-    ```
+For detailed explanation of each field, please refer to [v2alpha1-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v2alpha1-reference.md)
 
-    Full example please check [`emqx-full.yaml`](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v2alpha1/emqx-full.yaml).
+2. Check whether the EMQX cluster is ready
 
-    Detailed explanation of each field please check [v2alpha1-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v2alpha1-reference.md)
+```bash
+kubectl get emqx emqx -o json | jq ".status.conditions"
+```
 
-2. Check EMQX Custom Resource status
+The output is similar to:
+
+```bash
+[
+  {
+    "lastTransitionTime": "2023-02-07T06:46:39Z",
+    "lastUpdateTime": "2023-02-07T06:46:39Z",
+    "message": "Updating core nodes in cluster",
+    "reason": "ClusterCoreUpdating",
+    "status": "True",
+    "type": "CoreNodesUpdating"
+  },
+  {
+    "lastTransitionTime": "2023-02-07T06:46:38Z",
+    "lastUpdateTime": "2023-02-07T06:46:38Z",
+    "message": "Creating EMQX cluster",
+    "reason": "ClusterCreating",
+    "status": "True",
+    "type": "Creating"
+  }
+]
 
-    ```
-    $ kubectl get pods
-    $ kubectl get emqx emqx -o json | jq ".status.emqxNodes"
-    ```
+```
 
 ### Deploy EMQX 4
 
-1. Deploy EMQX Custom Resource
+1. Deploy EMQX
+
+```bash
+cat << "EOF" | kubectl apply -f -
+apiVersion: apps.emqx.io/v1beta4
+kind: EmqxBroker
+metadata:
+   name: emqx
+spec:
+   template:
+     spec:
+       emqxContainer:
+         image:
+           repository: emqx/emqx-ee
+           version: 4.4.14
+EOF
+```
 
-    ```bash
-    cat << "EOF" | kubectl apply -f -
-      apiVersion: apps.emqx.io/v1beta3
-      kind: EmqxBroker
-      metadata:
-        name: emqx
-      spec:
-        emqxTemplate:
-          image: emqx/emqx:4.4.9
-    EOF
-    ```
+For a complete example, please see [emqxbroker-full.yaml](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v1beta4/emqxbroker-full.yaml)
 
-    Full example please check [`emqxbroker-full.yaml`](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v1beta3/emqxbroker-full.yaml).
+For detailed explanation of each field, please refer to [v1beta4-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta4-reference.md)
 
-    Detailed explanation of each field please check [v1beta3-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta3-reference.md)
+2. Check whether the EMQX cluster is ready
 
-2. Check EMQX Custom Resource status
+```bash
+kubectl get emqxbroker emqx -o json | jq ".status.conditions"
+```
 
-    ```
-    $ kubectl get pods
-    $ kubectl get emqxbroker emqx -o json | jq ".status.emqxNodes"
-    ```
+The output is similar to:
```

Comment:
Make sure the item has `type ：Running` and `status：True`

Reply:
> Make sure the item has `type ：Running` and `status：True`

OK，I know you mean